.github/workflows/node.yml

```diff
@@ -181,8 +207,19 @@ jobs:
 
             - name: Test compatibility
               run: npm test -- -t "set and get flow works"
-              working-directory: ./node                   
-        
+              working-directory: ./node
+
+            - name: Upload test reports
+              if: always()
+              continue-on-error: true
+              uses: actions/upload-artifact@v4
+              with:
+                  name: test-report-node-amazonlinux
+                  path: |
+                      node/test-report*.html
+                      utils/clusters/**
+                      benchmarks/results/**
```

Comment:
Is there no way to reduce the duplication of this step? It's going to be a pain to remember to update all instances if we change it later.

Reply:
I afraid no - if I move it to another job, which depends on others - it will be executed on an empty VM.

I can move to a common action (like we have "Install shared dependencies"), but still we have to trigger it from there. It won't save much duplicated code.

Reply:
:/ unfortunate, but okay.